### PR TITLE
lipper1110 proposal submission

### PR DESCRIPTION
**Pull request by lipper1110**
---
A good PR should include the following parts:

- [x] A clear title
- [x] A meaningful message for PR, as well as its commits
- [x] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [x] excluding any irrelevant files, such as binaries, text files, or dot files
- [x] passing tests/CI

### CODE REVIEW

File: project/lipper1110/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+## Prospective Users
+Financial institutions trader, hedge funds, retail investors, people who want to make money from the secondary market
+
+## System Architecture
+![](https://i.imgur.com/BK5h5LN.png)
+
+## API Description
+### Python
+* `crawler`
```
Comment (yungyuc) [inline project/lipper1110/README.md:39]: The plan looks complete.  Crawling could take some time.  I do not know how much experience you have with crawling.  If not a lot, limiting the scope of data could be an option.

**Review by yungyuc — COMMENTED** [review]
Please leave a message to let me know when you want this to be merged.

File: project/lipper1110/README.md (hunk 1 above)
Comment (lipper1110) [reply project/lipper1110/README.md:39]: Thanks, maybe I will crawl the market data for 5 years.

**Comment by lipper1110** [comment]
> Please leave a message to let me know when you want this to be merged.

I think the proposal can be merged, thanks.

**Pull request merged**